starboard/nplb/maximum_player_configuration_explorer.cc

```diff
@@ -0,0 +1,221 @@
+// Copyright 2023 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "starboard/nplb/maximum_player_configuration_explorer.h"
+
+#include <algorithm>
+#include <deque>
+#include <unordered_map>
+
+#include "starboard/common/log.h"
+#include "starboard/nplb/player_test_util.h"
+#include "starboard/player.h"
+
+namespace starboard {
+namespace nplb {
+
+namespace {
+
+class HashFunction {
+ public:
+  size_t operator()(const std::vector<int>& v) const {
+    size_t r = 0;
+    for (const auto& it : v) {
```

Comment:
Consider renaming `it` as it's not an iterator.

Reply:
done